enhancements/node-latency-profile/node-latency-profiles.md

```diff
@@ -0,0 +1,276 @@
+---
+title: kubelet-cm-communication-profiles
+authors:
+  - "@harche"
+reviewers:
+  - "@rphillips, @sttts, @soltysh"
+approvers:
+  - "@rphillips, @sttts, @soltysh"
+creation-date: 2021-09-28
+last-updated: 2021-09-28
+status: implementable
+see-also:
+  - https://github.com/kubernetes-sigs/kubespray/blob/master/docs/kubernetes-reliability.md
+replaces:
+superseded-by:
+---
+
+# NodeLatencyProfile Profiles
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+
+## Summary
+
+To make sure Openshift cluster keeps running optimally under various circumstances where network latency, API Server latency, etcd latency, control plane load latency, etc. may not always be at their best (e.g. Edge cases), we can tweak the frequency of the status updates done by the `Kubelet` and the corresponding reaction times of the `Controller Manager` as well as `Kube API Server`.
+
+## Motivation
+
+In order to adjust to different latencies the cluster may experience, we need to adjust how frequently the `Kubelet` updates it's status and how do components such as `Controller Manager` and `Kube API Server` react to those status updates.
```

Comment:
how does KAS react to node updates? (other than maybe updating the host network endpoints?)

Reply:
Please refer to the section `Coordination between MCO, KCMO, CKAO` which talks in detail about flags and coordination between these components.